### PR TITLE
Setup basic HTTP wrapper module

### LINKED ISSUE #27: Utility for HTTP Requests
## Description

- [ ] Create a utility for http requests.

### PR DESCRIPTION
**Pull request by sbimochan**
Wrapper of `requests`. HTTP for users

resolves #27

### CODE REVIEW

File: datamegh/api/http.py [hunk 1]
```diff
@@ -1 +1,38 @@
-""" HTTP Connection API. """
+""" Basic HTTP verbs for your API connection. """
+
+import requests
+
+
+def get(url, params=None, **kwargs):
```
Comment (kabirbaidhya) [inline datamegh/api/http.py]: Doc strings. Every function requires doc strings. 
```suggestion
def get(url, params=None, **kwargs):
    """ Sends an HTTP GET request. """
```

Comment (kabirbaidhya) [reply datamegh/api/http.py]: Same for every other functions.

File: datamegh/api/http.py [hunk 2]
```diff
@@ -1 +1,38 @@
...
+    response = requests.get(url, params, **kwargs)
+    return response
```
Comment (kabirbaidhya) [inline datamegh/api/http.py]: You can directly return as there aren't any intermediate steps here.  
```suggestion
    return requests.get(url, params, **kwargs)
```

File: tests/api/http/test_http.py [hunk 3]
```diff
@@ -0,0 +1,54 @@
+import pytest
+
+from datamegh.api import http
+
+TEST_URL = "https://httpbin.org/"
+
+
+def test_get():
```
Comment (kabirbaidhya) [inline tests/api/http/test_http.py]: docstring same as above.

**Comment by kabirbaidhya** [comment]
@sbimochan Can you add a the custom verb method too? 
https://requests.readthedocs.io/en/master/user/advanced/#custom-verbs

**Review by kabirbaidhya — APPROVED** [review]

**Comment by praneen** [comment]
> @sbimochan Can you add a the custom verb method too?
> https://requests.readthedocs.io/en/master/user/advanced/#custom-verbs

Didn't know such custom verbs could be used too. :)

**Pull request merged**